README.md

```diff
@@ -1 +1,32 @@
-# Assignment
+# Commenting and Documentation Standards
```

Comment:
We use ### for these headings I believe, maybe check the master file to be sure

Reply:
changed